test/functional/wallet_bumpfee.py

```diff
@@ -375,21 +346,6 @@ def test_watchonly_psbt(self, peer_node, rbf_node, dest_address):
     rbf_node.unloadwallet("watcher")
     rbf_node.unloadwallet("signer")
 
-def test_rebumping(self, rbf_node, dest_address):
-    self.log.info('Test that re-bumping the original tx fails, but bumping successor works')
-    rbfid = spend_one_input(rbf_node, dest_address)
-    bumped = rbf_node.bumpfee(rbfid, {"totalFee": 2000})
-    assert_raises_rpc_error(-4, "already bumped", rbf_node.bumpfee, rbfid, {"totalFee": 3000})
-    rbf_node.bumpfee(bumped["txid"], {"totalFee": 3000})
-
-
-def test_rebumping_not_replaceable(self, rbf_node, dest_address):
```

Comment:
this seems easily updateable to feerate and worthwhile to check?

Reply:
thanks -- done